InteractionBase/src/handlers/GenericHandler.cpp

```diff
@@ -510,6 +514,33 @@ void GenericHandler::keyPressEvent(Visualization::Item *target, QKeyEvent *event
 
 		scene->scheduleUpdate();
 	}
+	else if (event->key() == Qt::Key_F12)
+	{
+		event->accept();
+		if (SelectionAtCursorItem::isVisible())
+			SelectionAtCursorItem::hide();
+		else
+		{
+			QVector<Model::Node*> nodes;
+			for (auto view : target->scene()->viewItems())
+				nodes.append(new Interaction::ViewSelectionNode(view->name()));
+
+			SelectionAtCursorItem::OnSelectAction onSelect =
```

Comment:
The reason I did this is so that this menu could be used for other things as well if needed, by simply passing different types of nodes and selection handling methods.